src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -13,7 +13,7 @@ import { getNextChunk, receiveDotNetDataStream } from './StreamingInterop';
 import { RootComponentsFunctions } from './Rendering/JSRootComponents';
 import { attachWebRendererInterop } from './Rendering/WebRendererInteropMethods';
 
-interface IBlazor {
+export interface IBlazor {
```

Comment:
I think you could use the `Blazor` const in your other function rather than `IBlazor`. TBH `IBlazor` doesn't serve any purpose and we should ideally eliminate it some time in the future.

Reply:
I'll give it a try

Reply:
This doesn't really work, since I needed it at a type position within the other file. I've chosen to type the functions as `unknown` since the only thing we do is pass through data

Reply:
Nevermind, I found the way